### PR TITLE
[Network] DestinationPortBehavior in Get-AzNetworkWatcherConnectionMonitor is not getting information with powershell command

### PR DESCRIPTION
**Pull request by v-sapanchal**
## Description

DestinationPortBehavior in Get-AzNetworkWatcherConnectionMonitor is not getting information with powershell command
https://github.com/Azure/azure-powershell/issues/15996

For example, If you run this command in powershell to get the connection monitor protocol configuration 
and pass the -**Name** parameter by replacing the **{connectionMonitorName}** with your connection monitor name.
`
(get-AzNetworkWatcherConnectionMonitor -NetworkWatcherName "NetworkWatcher_centraluseuap" -ResourceGroupName NetworkWatcherRg -Name "**{connectionMonitorName}**").TestGroups[0].TestConfigurations.ProtocolConfiguration
`

It displays DestinationPortBehavior value as **blank**.

Port                    : 80
DisableTraceRoute       : True
**DestinationPortBehavior** :    

`Note : Make sure connect with your AzAccount `

It should be displayed as "**ListenIfAvailable**" with PowerShell command,  if "Listen on port" checkbox is checked ( you can verify this checkbox on "portal.azure.com",
Home>Network Watcher>Connection Monitor>{**connectionMonitorName**}>Edit Connection Monitor> Edit Test configurations toggle, Here you can see the "Listen on port" checkbox (if Protocol is TCP).
`Note : DestinationPortBehavior value can be blank if "Listen on port" checkbox is unchecked.`

After this fix, It will display like,

Port                    : 80
DisableTraceRoute       : True
DestinationPortBehavior : **ListenIfAvailable**

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by msJinLei** [comment]
@v-sapanchal Please description your change in the changelog.md
Please ask your team member to review your change.

File: src/Network/Network/ChangeLog.md [hunk 1]
```diff
@@ -23,6 +23,9 @@
 * Added `DdosProtectionPlan` property in `AzPublicIpAddress`
 * Updated mapping in `AzPublicIpAddress` to always show/create DdosSettings
 * Fixed a bug that added Ddos related properties when viewing PublicIpAddress and DdosProtectionPlan objects
+* Fixed a Bug for Set-AzIpGroup cmdlet to support the -whatif parameter
```
Comment (kumaam) [inline src/Network/Network/ChangeLog.md:26]: Redundant lines got added after resolving merge-conflicts. Please check

**Review by dingmeng-xue — DISMISSED** [review]

File: src/Network/Network/ChangeLog.md (hunk 1 above)
Comment (v-sapanchal) [reply src/Network/Network/ChangeLog.md]: removed the redundant lines

**Comment by dingmeng-xue** [comment]
@v-sapanchal , you should not merge main to your branch. Could you clean your branch? (remove your last commit)

**Comment by dingmeng-xue** [comment]
Please close this pull request because #20578 is merged.

**Pull request closed without merging**